src/components/Auth/AuthContainer.js

```diff
@@ -26,25 +42,31 @@ class AuthContainer extends React.Component {
           studentId: data.studentID,
           role: data.role
         })
-          .then(() => {
-            console.log("User was registered");
+      })
+      .then(() => {
+        console.log("User was registered");
```

Comment:
Just started seriously looking in linters. Probably going to be added soon. To start, this should be single-quotes not double.